egs/librispeech/ASR/prepare.sh

```diff
@@ -19,8 +18,53 @@ fi
 if [ $stage -le 0 ] && [ $stop_stage -ge 0 ]; then
   echo "stage 0: Download data"
 
-  # If you have pre-downloaded it in /path/to/LibriSpeech
-  # Just run: ln -sfv /path/to/LibriSpeech data/
+  # If you have pre-downloaded it to /path/to/LibriSpeech,
+  # you can create a symlink to avoid downloading it again:
+  #
+  #   ln -sfv /path/to/LibriSpeech data/
+  #
+
   mkdir -p data/LibriSpeech
-  # TODO
+
+  if [ ! -f data/LibriSpeech/train-other-500/.completed ]; then
```

Comment:
I don't think that this check is technically needed; i.e. the download script would figure the completion status by itself

OTOH I am concerned about fixing the location of the dataset in `data/LibriSpeech` -- people tend to have corpora downloaded in some standard locations on their own setups, I think this should remain customizable

Reply:
> the download script would figure the completion status by itself

If the zipped files are removed after extraction, the script will download it again. That's why I add this check to avoid invoking the download script.


>  people tend to have corpora downloaded in some standard locations on their own setups, I think this should remain customizable

Fix the location of the dataset makes the code simpler. Can we let the users create a symlink to its original dataset path, i.e., 
```
ln -s /path/to/LibriSpeech data/
```
as mentioned in the script
https://github.com/k2-fsa/icefall/blob/0b19aa09c1d1921affae37688115ebfdb090b188/egs/librispeech/ASR/prepare.sh#L21-L25

Reply:
ah, I see, I missed the bit about the symbolic link before. I am OK with that. 

> If the zipped files are removed after extraction, the script will download it again. That's why I add this check to avoid invoking the download script.

I see... maybe this is the right way then. I'm not sure if there is a straightforward way to address this issue inside of `lhotse download`.

Reply:
... I take that back -- I think we can change Lhotse so that the "completed detector" is executed before downloading the files ([move this line](https://github.com/lhotse-speech/lhotse/blob/master/lhotse/recipes/librispeech.py#L58) a few lines up)

I can make this change later for all the recipes. WDYT @csukuangfj ?

Reply:
> I can make this change later for all the recipes.

That's would be great. In that case, I won't need to add that check here.